cluster/config-defaults.yaml

```diff
@@ -120,14 +120,12 @@ skipper_ingress_tokeninfo_cpu: "1000m"
 skipper_ingress_tokeninfo_memory: "512Mi"
 {{if eq .Environment "production"}}
 tokeninfo_url: "http://127.0.0.1:9021/oauth2/tokeninfo"
-skipper_local_tokeninfo: "true"
-skipper_local_sandbox_tokeninfo: "false"          # TODO(sszuecs): delete CR entries
-skipper_local_sandbox_bridge: "false"             # TODO(sszuecs): delete CR entries
```

Comment:
I dropped these from all clusters where they were set in CR (they were all set to the default so not functional change)

Reply:
thanks for the cleanup